stacks/core/README.md

```diff
@@ -33,12 +46,30 @@ This stack will be deployed with the basic services, which are related to the th
    knot-cloud create-user <email> <password> --server api.knot.cloud --protocol https
    ```
 
-1. Create a new user account in the **KNoT Fog** and save the output:
+   ```bash
+   knot-cloud create-user knot@cesar.org.br exemplo-senha --server api.knot.cloud --protocol https
+
+   user successfully created
+   {
+     "token": "some-token-returned-from-cloud"
+   }
+   ```
+
+1. Create a new user account in the **KNoT Fog** and save the output (you can use the same e-mail and password from the previous command):
 
    ```bash
    knot-cloud create-user <email> <password> --server api.fog --protocol http
    ```
 
+   ```bash
+   knot-cloud create-user knot@cesar.org.br exemplo-senha --server api.knot.cloud --protocol https
```

Comment:
I believe we should either keep just one of these commands or make it clear the second one is an example, I think this, as it is now, looks a bit confusing on [the rendered document](https://github.com/netoax/knot-cloud/blob/devel_fix_readme/stacks/core/README.md#Development).

Reply:
Agreed.

Reply:
Done. Could you take a look?